client/src/components/application/sections/StdHeader.vue

```diff
@@ -66,8 +66,7 @@
                 </div>
               </div>
             </v-form>
-            <div id="userid" class="ml-1 mt-auto mb-auto fv-ital"
-                             :class="wordClassificationFlag ? 'mr-5 pr-5' : ''">{{ userId }}</div>
```

Comment:
this feature flag should have been removed entirely per the requirements of a previous entity ticket. since the feature is now permanently on, i consolidated the classes from the `true` side of the dynamic `:class` ternary into the static classes.